index.html

```diff
@@ -6,15 +6,181 @@
+      <div class="cards d-flex flex-column flex-md-row gap-3">
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+        <div class="card bg-semi-trans d-flex flex-row flex-md-column gap-3 align-items-center p-3">
+          <img class="w-30" src="images/icons/lecture.svg" alt="">
+          <h3 class="text-bold accent mb-0">Lecture</h3>
+          <p class="text-light mb-0">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt incidunt debitis.
+          </p>
+        </div>
+      </div>
+      <div class="footer d-flex mt-5">
+        <button type="button" class="btn btn-custom text-light mx-auto p-3 bg-accent">Join Conference 2021</button>
+        <a class="d-none text-center" href="#">Learn More Information</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="speakers py-5">
+    <div class="wrapper container-sm">
+      <h2 class="heading text-dark text-center text-capitalize mb-5">featured speakers</h2>
+      <div class="d-grid gap-3">
+        <div class="row">
+          <div class="col-left col-sm-12 col-md-12 col-lg-6 d-flex flex-column gap-3 mb-3">
+          </div>
+          <div class="col-right col-sm-12 col-md-12 col-lg-6 d-flex flex-column gap-3">
+          </div>
+        </div>
+      </div>
+      <div class="footer d-md-none d-flex mt-5">
+        <a class="btn btn-custom text-capitalize text-light mx-auto p-3 bg-accent">more speakers</a>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] On this section, I suggest you should remove the borders around the divs and add some background behind the images.